app/views/draws/show.html.erb

```diff
@@ -69,7 +69,7 @@
   <% if policy(@draw).lottery? %>
     <%= link_to 'Assign lottery numbers', lottery_draw_path(@draw), class: 'button secondary' %>
   <% end %>
-  <% if policy(Group).new? %>
+  <% if policy(Group).new? && policy(@draw).group_actions? %>
```

Comment:
There's no way to pass the `@draw` variable to the GroupPolicy for `new?`, since there's no actual group involved (it authorizes the class). I looked into this and it is strongly discouraged anyway, see [here](https://github.com/elabs/pundit#additional-context). I know it's not great but I'm not sure there's an alternative. Let me know what you think!